bin/es_alias_delete

```diff
@@ -5,7 +5,7 @@ require "datura"
 require "json"
 require "rest-client"
 
-params = Datura::Parser.es_alias_add
+params = Datura::Parser.es_alias_delete
```

Comment:
I had seen this, but I assumed they used the exact same options etc and you were just sharing :)

Reply:
I think that might have been what I was doing at the beginning?  I can probably change it so that they both use the same options but since I already had a parser file for deletion I just switched it to using that this time around.

Reply:
Oh it's fine this way. It might be harder to communicate that it's a shared parser with the parser's name, so having separate ones might be safer.